get_models_and_test.py

```diff
@@ -4,6 +4,8 @@
 from data_generator import OutageData
 from outage_loss import InfiniteOutageCoefficientLoss, TPR, FPR, Precision
 import toy_models
+import dqn_lstm
```

Comment:
This doesn't look right. It looks like you've imported this and then assigned an instance of the DQNLSTM class to it on line 124

Reply:
dqn_lstm.py is the file. On line 124, dqn_lstm is the variable which is an instance of the DQNLSTM class. But i see the issue, they are named the same, and it creates confusion. I will rename the variable at line 124.

Reply:
Since you're note using the `import dqn_lstm` (you're just using `from dqn_lstm import DQNLSTM`), can you remove it?

Reply:
Done.